Pod/Classes/WPMediaPickerViewController.m

```diff
@@ -1455,34 +1519,21 @@ - (void)keyboardWillHideNotification:(NSNotification *)notification
     contentInset.bottom = 0.f;
     self.collectionView.contentInset = contentInset;
     self.collectionView.scrollIndicatorInsets = contentInset;
-    [self.emptyViewBottomConstraint setConstant:0.f];
+    self.currentKeyboardHeight = 0.f;
+    [self.emptyViewBottomConstraint setConstant:self.currentKeyboardHeight];
 
     [UIView animateWithDuration:0.2 animations:^{
-        [self centerEmptyView];
+        [self adjustEmptyView];
         [self.collectionView.collectionViewLayout invalidateLayout];
     }];
 }
 
 /**
- Centers the empty view taking into account the collection view height and content insets.
+ Adjusts empty view to refresh constraints based on the main view.
  */
-- (void)centerEmptyView
+- (void)adjustEmptyView
```

Comment:
This function name implies that we're doing more than just refreshing the layout. I'd consider removing it and just calling `[self.view layoutIfNeeded]` directly when needed.

Reply:
Resolved here https://github.com/wordpress-mobile/MediaPicker-iOS/commit/e54f6a1a818e1226233fcbca6bb9649e57447b42 😄